third_party/move/tools/move-spec-test/src/report.rs

```diff
@@ -0,0 +1,202 @@
+
+        builder.push_record(["File", "Mutants tested", "Mutants killed", "Percentage"]);
+
+        for (path, stats) in self.files.iter() {
+            builder.push_record([
+                path.to_string_lossy().to_string(),
+                stats.tested.to_string(),
+                stats.killed.to_string(),
+                format!(
+                    "{:.2}%",
+                    (stats.killed as f64 / stats.tested as f64) * 100.0
+                ),
+            ]);
+        }
+
+        let table = builder.build().with(Style::modern_rounded()).to_string();
+
+        println!("{}", table);
+    }
+}
+
+/// This struct represents an entry in the report.
+/// It contains the number of mutants tested and killed.
+#[derive(Default, Debug, Serialize)]
+pub struct MutantStats {
+    /// The number of mutants tested.
+    pub tested: u64,
+    /// The number of mutants killed.
+    pub killed: u64,
+}
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use std::path::PathBuf;
+
+    #[test]
+    fn report_starts_empty() {
+        let report = Report::new();
+        assert_eq!(report.entries().len(), 0);
+    }
+
+    #[test]
+    fn adding_entry_increases_count() {
+        let mut report = Report::new();
+        let path = PathBuf::from("path/to/file");
+        let stats = MutantStats {
+            tested: 5,
+            killed: 3,
+        };
+        report.add_entry(path, stats);
+        assert_eq!(report.entries().len(), 1);
+    }
+
+    #[test]
+    fn contains_returns_true_for_existing_entry() {
+        let mut report = Report::new();
+        let path = PathBuf::from("path/to/file");
+        let stats = MutantStats {
+            tested: 5,
+            killed: 3,
+        };
+        report.add_entry(path.clone(), stats);
+        assert!(report.contains(&path));
+    }
+
+    #[test]
+    fn contains_returns_false_for_non_existing_entry() {
+        let report = Report::new();
+        let path = PathBuf::from("path/to/file");
+        assert!(!report.contains(&path));
+    }
+
+    #[test]
+    fn increment_mutants_tested_increases_tested_count() {
+        let mut report = Report::new();
+        let path = PathBuf::from("path/to/file");
+        report.increment_mutants_tested(&path);
+        assert_eq!(report.entries().get(&path).unwrap().tested, 1);
+    }
+
+    #[test]
+    fn increment_mutants_killed_increases_killed_count() {
+        let mut report = Report::new();
+        let path = PathBuf::from("path/to/file");
+        report.increment_mutants_killed(&path);
+        assert_eq!(report.entries().get(&path).unwrap().killed, 1);
+    }
+
+    #[test]
+    fn mutants_tested_returns_total_tested_count() {
+        let mut report = Report::new();
+        let path1 = PathBuf::from("path/to/file1");
+        let path2 = PathBuf::from("path/to/file2");
+        report.increment_mutants_tested(&path1);
+        report.increment_mutants_tested(&path2);
+        assert_eq!(report.mutants_tested(), 2);
+    }
```

Comment:
A bit of extra test context:
```suggestion
        assert_eq!(report.mutants_tested(), 2);
        assert_eq!(report.mutants_killed(), 0);
    }
```

Reply:
Added.